### PR TITLE
Footer for About Screen of React Native mobile app

### LINKED ISSUE #3203: Adding footer for mobile app's about screen
**What would you like to be added**:
After this [PR](https://github.com/Seneca-CDOT/telescope/pull/3167) gets merger, a footer needs to be added to the about screen of our React Native app.
image
**Why would you like this to be added**:
Because of the original design of Telescope

### PR DESCRIPTION
**Pull request by nguyenhung15913**
## Issue This PR Addresses
Fixes #3203 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [x] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description
- Added Footer for about screen
- Github API was not added so the contributor card still remains static

## Screenshot
image

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by tpmai22** [comment]
So the GitHub contribution card is just a placeholder for now ?

**Comment by nguyenhung15913** [comment]
> So the GitHub contribution card is just a placeholder for now ?

yeah. I think we should create another issue for adding github api to the footer. since it's a different feature.

**Comment by tpmai22** [comment]
@nguyenhung15913 let file it and I will aprrove this PR

File: src/mobile/src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+      <Image style={styles.stretch} source={require('../assets/adaptive-icon.png')} />
+      <View style={styles.column}>
+        <View style={styles.row}>
+          <View style={styles.left}>
+            <Text style={styles.heading}>Docs</Text>
+            <View style={styles.leftDivider}></View>
+            <Pressable
+              onPress={() =>
+                Linking.openURL(
+                  'https://github.com/Seneca-CDOT/telescope/blob/master/docs/environment-setup.md'
```
Comment (JiaHua-Zou) [inline src/mobile/src/components/Footer.jsx:80]: The link to this page is not found. Did you mean this link? `https://github.com/Seneca-CDOT/telescope/blob/master/src/docs/docs/getting-started/environment-setup.md`

File: src/mobile/src/components/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                )
+              }
+            >
+              <Text style={styles.subheading}>Get Started</Text>
+            </Pressable>
+            <Pressable
+              onPress={() =>
+                Linking.openURL(
+                  'https://github.com/Seneca-CDOT/telescope/blob/master/docs/CONTRIBUTING.md'
```
Comment (JiaHua-Zou) [inline src/mobile/src/components/Footer.jsx:89]: same here link not found. Did you mean this one? `https://github.com/Seneca-CDOT/telescope/blob/master/src/docs/docs/contributing/CONTRIBUTING.md`

Comment (nguyenhung15913) [reply src/mobile/src/components/Footer.jsx]: done

File: src/mobile/src/components/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+      <Image style={styles.stretch} source={require('../assets/adaptive-icon.png')} />
+      <View style={styles.column}>
+        <View style={styles.row}>
+          <View style={styles.left}>
+            <Text style={styles.heading}>Docs</Text>
+            <View style={styles.leftDivider} />
+            <Pressable
+              onPress={() =>
+                Linking.openURL(
+                  'https://github.com/Seneca-CDOT/telescope/blob/master/src/docs/docs/getting-started/environment-setup.md'
```
Comment (humphd) [inline src/mobile/src/components/Footer.jsx:79]: We should file a follow-up issue to use the main entry point for our docs site, since this might change.

**Review by humphd — DISMISSED** [review]

**Review by JiaHua-Zou — DISMISSED** [review]

**Review by Kevan-Y — DISMISSED** [review]

**Review by tpmai22 — APPROVED** [review]

**Review by TueeNguyen — APPROVED** [review]

**Pull request merged**